### ISSUE TITLE
v3.0 moving method to package then reloading project gets 'Duplicate package' error

### ISSUE THREAD

**Issue by ericwinger** [GemTalk/Rowan#803]
Project: RowanClientServices sha: 84f65de

Reproduction case: 
- Load RowanClientServices package with sha indicated (may reproduce for other projects though)
- Move method `addPackageNamed:toComponentNamed:` from `Rowan-Services-CoreV2` package to `Rowan-Services-Core` package.  
- In Jadeite, this will dirty the RowanClientServices package. 
- Reload the project from Jadeite or by sending `reload` to the appropriate RwProject. 
- Get duplicate package error. 

Stack is attached. 

Note - I haven't seen this error moving methods around. It's worth noting that we're moving the last extension method out of a package.  _(Edit - Note that I could reproduce this without moving the last extension method out of the package)_ 

[20220316_duplicate_package_error.txt](https://github.com/GemTalk/Rowan/files/8273823/20220316_duplicate_package_error.txt)

**Comment by ericwinger** [comment]
Note that I just got this again without (seemingly) doing anything to the component structure. I just pressed 'Changes' for the project and got this stack. 
[20220329_duplicate_package_rowan_stack.txt](https://github.com/GemTalk/Rowan/files/8375581/20220329_duplicate_package_rowan_stack.txt)

**Comment by dalehenrich** [comment]
1. how do you move the method between packages? Are you dragging the method to a different category or are you using an API call
2. In the second case (reported yesterday), is that in the same image that you were using for the first case, or? I assume that 'Changes' has been working at some point in time..
3. Do I need to have a new Jadeite to run with the new RowanClientServices?

**Comment by ericwinger** [comment]
>     how do you move the method between packages? Are you dragging the method to a different category or are you using an API call

I believe I dragged the method to a new package in the new Jadeite. This ultimately sends `rwMoveMethod:toPackage:` to the class. 

>     In the second case (reported yesterday), is that in the same image that you were using for the first case, or? I assume that 'Changes' has been working at some point in time..

Yes, and it is true that I hadn't pressed the menu option `Changes` recently. It's not inconceivable that the problem has been lurking in my image for some time.  However, I was able to work around the original issue report by aborting my image.  

FWIW - I don't recall moving methods to new packages between the aforementioned abort and pressing `Changes`. 

>     Do I need to have a new Jadeite to run with the new RowanClientServices?

I don't *think* so, but if you want to replicate the drag/drop move method to package function exactly as I described, I would need to cut a new Jadeite.
